gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/apim/core/api/use_case/MigrateApiUseCaseTest.java

```diff
@@ -1583,159 +1675,6 @@ void should_migrate_api_with_http2_endpoint_configuration() {
             });
     }
 
-    @Nested
-    class FlowSpecific {
```

Comment:
I'm wondering you made a misakes where rebasing it seems you remove the work done by Michel (https://github.com/gravitee-io/gravitee-api-management/pull/13022)